core/src/test/scala/org/apache/spark/io/CompressionCodecSuite.scala

```diff
@@ -128,6 +130,69 @@ class CompressionCodecSuite extends SparkFunSuite {
     }
   }
 
+  test("SPARK-25895 Zstd shuffle Read/Write/spill comparison w.r.t lz4") {
```

Comment:
How about add a benchmark? You can check our all benchmarks here:: https://issues.apache.org/jira/browse/SPARK-25475

Reply:
Thanks for your suggestion, i will go through the benchmarks and update the PR